engine/src/main/scala/cromwell/webservice/routes/CromwellApiService.scala

```diff
@@ -186,7 +185,23 @@ trait CromwellApiService extends HttpInstrumentation with MetadataRouteSupport w
     WorkflowSubmitResponse(workflowId.toString, workflowState.toString)
   }
 
-  private def submitRequest(formData: Multipart.FormData, isSingleSubmission: Boolean): Route = {
+  def standardSuccessHandler: PartialFunction[WorkflowStoreSubmitActorResponse, Route] = {
+    case WorkflowStoreSubmitActor.WorkflowSubmittedToStore(workflowId, _) => complete(WorkflowSubmitResponse(workflowId.toString, WorkflowSubmitted.toString))
+    case WorkflowStoreSubmitActor.WorkflowsBatchSubmittedToStore(workflowIds, _) => complete(workflowIds.toList.map(x => WorkflowSubmitResponse(x.toString, WorkflowSubmitted.toString)))
```

Comment:
Looks like this is where the 200 vs 201 test failure is coming from. FWIW I think WES should also allow a 201 for newly created workflows so it would be fine to return that here.